src/main/java/seedu/exercise/model/exercise/Property.java

```diff
@@ -1,4 +1,4 @@
-package seedu.address.model.exercise;
+package seedu.exercise.model.exercise;
 
 /**
  * Represents a property of an {@code Exercise} object.
```

Comment:
Is there a better way to make sure each individual property implement the following methods? For @weihaw08 to consider. Cos it seems like the Property abstract class is just forcing child classes to override the Object methods.